### PR TITLE
Proposing addition of two new toast types: "info" and "warning"

### PR DESCRIPTION
**Pull request by zicho**
This PR includes two new types of toasts, "info" and "warning". 

### Icons
I have not yet added any icons for these toasts because I did not know the source of your SVG. I can easily fix it with any MIT licensed SVG though, but I would prefer it to be consistent with your design

### Colors
I just tweaked the existing colors to a shade of blue and yellow. Open to change them as well but they look decent imo.

### CODE REVIEW

**Review by wobsoriano — CHANGES_REQUESTED** [review]
Minor request, everything looks good.

Can you pull the main updates and run `pnpm lint --fix`? This formats it to use single quotes. Thanks!

File: src/lib/Icon.svelte [hunk 1]
```diff
@@ -1,19 +1,59 @@
...
+    fill="currentColor"
+    height="20"
+    width="20"
+  >
+    <path
+      fill-rule="evenodd"
+      d="M18 10a8 8 0 11-16 0 8 8 0 0116 0zm-8-5a.75.75 0 01.75.75v4.5a.75.75 0 01-1.5 0v-4.5A.75.75 0 0110 5zm0 10a1 1 0 100-2 1 1 0 000 2z"
+      clip-rule="evenodd"
+    />
+  </svg>
```
[the remark this answers is not in the document]
Comment (zicho) [reply src/lib/Icon.svelte]: It's MIT so it should be all good. I added this to my latest commits. Error and warning icon are still identical, though

**Comment by zicho** [comment]
I have merged with changes in main and run the linter to address the formatting issues.

File: src/lib/Icon.svelte (hunk 1 above)
[the remark this answers is not in the document]
Comment (wobsoriano) [reply src/lib/Icon.svelte]: Thanks. Yeah, we can keep the error icon for now.

**Review by wobsoriano — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**